code/controllers/subsystems/inventory.dm

```diff
@@ -0,0 +1,24 @@
+SUBSYSTEM_DEF(inventory)
```

Comment:
What the heck do we need an inventory subsystem for?

Reply:
It's not fire or smth like that.
I'm not sure is it really reqed but that seems like proper way to initialize slot datums.
And that's easier to debug SubSystem when global list of slots.